Azure/azure-cli
# [AKS] `az aks create/update`: Add new parameter `--node-os-upgrade-channel` to specify which OS on your nodes is updated

…com/Azure/azure-cli-extensions/pull/5788/files

**Related command**
az aks create|update

**Description**
This allows the setting of node os upgrade channel in create and update of aks clusters.
Public docs are here. https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-node-image

It is GAing this preview extension: https://github.com/Azure/azure-cli-extensions/pull/5788

This became available in the 06-01 api.

**Testing Guide**
Same tests as extension create and update with nodeos upgrade channel. Make sure its preserved. Combine with upgrade chennel

**History Notes**
[AKS] `az aks create/update`: Add new parameter `--node-os-upgrade-channel` to specify which OS on your nodes is updated

---

This checklist is used to make sure that common guidelines for a pull request are followed.
